src/fluids.md

```diff
@@ -0,0 +1,184 @@
+# Navier-Stokes Mini Application
+
+The solver implemented in this miniapp solves the transient incompressible
+Navier-Stokes equations.
+
+## Theory
+
+The equations are given in the non-dimensionalized form
+
+\begin{align}
+    \frac{\partial u}{\partial t} + (u \cdot \nabla) u - \frac{1}{Re} \nabla^2 u + \nabla p &= f & \quad \text{in } \Omega\\\\
+    \nabla \cdot u &= 0 & \quad \text{in } \Omega
+\end{align}
+
+where $Re$ represents the Reynolds number. In order to solve these equations,
+the method presented in Tomboulides (1997)[^1] is used, which is based on an
+equal order finite element discretization on quadrilateral or hexahedral
+elements of high polynomial order. The method describes an implicit-explicit
+time-integration scheme for the viscous and convective terms respectively.
+Introducing the following notation for the linear term $L(u) = \frac{1}{Re}
+\nabla u$ and the nonlinear term $N(u) = -(u \cdot \nabla) u$ and their
+time-extrapolated forms
+
+\begin{align}
+    L^\*(u^{n+1}) = \sum_{j=1}^k a_j L(u^{n+1-j}) \\\\
+    N^\*(u^{n+1}) = \sum_{j=1}^k a_j N(u^{n+1-j})
+\end{align}
+
+where $a_j$ are coefficients from the corresponding explicit time integration
+method.
+
+Applying a BDF method to the initial equation using the introduced forms yields
+
+\begin{align}
+    \sum_{j=0}^k \frac{b_j}{\Delta t} u^{n+1-j} =
```

Comment:
Perhaps this is obvious but we could mention that the b_j are coefficients from the BDF method similar to how the a_j were declared above.